src/app/content/practiceQuestions/components/Answer/index.tsx

```diff
@@ -81,48 +81,52 @@ const Answer = ({
     }
   };
 
-  return <AnswerBlock
-    showCorrect={showCorrect}
-    isCorrect={isCorrect}
-    isSubmitted={isSubmitted}
-    isSelected={isSelected}
+  return <label
```

Comment:
You are nesting `label` inside a `label` right now.

Reply:
fixed